rotkehlchen/accounting/export/csv.py

```diff
@@ -331,3 +340,20 @@ def export(
             return False, str(e)
 
         return True, ''
+
+    def export_history(
+            self,
+            events: list['HistoryBaseEntry'],
+            directory: Path,
+    ) -> tuple[bool, str]:
+        serialized_events = [self.to_csv_entry(x) for idx, x in enumerate(events)]
```

Comment:
As mentioned in discord this should export everything not only the current PnL csv format for the history events https://discord.com/channels/657906918408585217/1080783409640648765/1155766596539777065

Reply:
Yes, I export everything for history events